src/components/settings/SettingsSSOTab/index.jsx

```diff
@@ -26,6 +29,9 @@ const SettingsSSOTab = ({ enterpriseId, setHasSSOConfig }) => {
         validConfigExists = true;
       }
     });
+    if (existingConfigs?.length < 1) {
```

Comment:
I'm a bit curious about the optional chaining construction here.  Do we want this expression to evaluate to false if existingConfigs is undefined or not an array-like object?

Reply:
to be honest this is just logic that is left over and I can add a null check but I feel like we can assume it's an array-like object